crates/bevy_reflect/bevy_reflect_derive/src/derive_data.rs

```diff
@@ -470,7 +470,12 @@ impl<'a> ReflectMeta<'a> {
         &self,
         where_clause_options: &WhereClauseOptions,
     ) -> proc_macro2::TokenStream {
-        crate::registration::impl_get_type_registration(self, where_clause_options, None)
+        crate::registration::impl_get_type_registration(
+            self,
+            where_clause_options,
+            None,
+            None::<std::iter::Empty<&Type>>,
```

Comment:
nit: i prefer the following:
```suggestion
            Option::<std::iter::Empty<&Type>>::None,
```